AmazonChimeSDKDemo/AmazonChimeSDKDemo.xcodeproj/project.pbxproj

```diff
@@ -789,7 +805,6 @@
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 0.16.6;
 				DEBUG_INFORMATION_FORMAT = dwarf;
-				DEVELOPMENT_TEAM = "";
```

Comment:
Why is this deleted?

Reply:
Probably Xcode deletes it after I reverting the test changes. I can add it back.